root/static/scripts/release-editor/fields.js

```diff
@@ -159,7 +159,7 @@ class Track {
 
     var newLength = utils.unformatTrackLength(length);
 
-    if (!newLength) {
+    if (Number.isNaN(newLength)) {
```

Comment:
This previously used https://lodash.com/docs/4.17.15#isNaN which is based on `Number.isNaN`, not `isNaN`:

> Note: This method is based on Number.isNaN and is not the same as global isNaN which returns true for undefined and other non-number values.